### DISCUSSION TITLE
How can I point to a shared models folder?

### DISCUSSION THREAD

**Discussion by KillavoltAI** [lllyasviel/stable-diffusion-webui-forge#697] [General]
I have A1111 installed as my "base" UI and there's where all the models/LORA's/etc are.
Is there a way to point ForgeUI to those folders so I don't have to duplicate all of the models?

**Comment by abzaloff** [comment]
Forge => webui => webui-user.bat - edit notepad. Write down your path. 
Example:
set COMMANDLINE_ARGS= --ckpt-dir "G:/Stable Diffusion/webui/models/Stable-diffusion" --lora-dir "G:/Stable Diffusion/webui/models/Lora"

**Reply by KillavoltAI** [reply]
I forgot to mention that I'm on Linux, but that being said I believe it's webui-user.sh. 
In any case, is there not a way to do this without using commandline args? Is there no settings or config file where this could be set?

**Reply by thePlott** [reply]
I believe you can set the path in webui-user.sh using --ckpt-dir /path/to/your/models/....... at the below location.

**# Commandline arguments for webui.py, for example: export COMMANDLINE_ARGS="--medvram --opt-split-attention"**

**Comment by RobertoMaurizzi** [comment]
I used `--forge-ref-a1111-home ../path/to/A1111` to point Forge to the same directories used by A1111 wholesale, since I did the same for ComfyUI

**Reply by Mystos007** [reply]
> I used `--forge-ref-a1111-home ../path/to/A1111` to point Forge to the same directories used by A1111 wholesale, since I did the same for ComfyUI

Thanks, I did that in Windows and it now works.

I thought something was up with how to do paths in windows because I was getting 'C:GitJunkA1111' in my error logs when I tried 'C:\GitJunk\A1111'.
